### ISSUE TITLE
SetupSequence is missing for protected members

### ISSUE THREAD

**Issue by s-limo** [moq/moq4#243]
Similar case works for public members but does not for protected:

m.Protected().SetupSequence("Member"()).Return(0).Return(1);

**Comment by paulomorgado** [comment]
I needed to do this:

```
stateMock.Protected()
    .Setup<Task<Transition>>("ExecuteStepAsync", It.IsAny<CancellationToken>())
    .Returns(nonTargetedTransitionTask)
    .Verifiable();
stateMock.Protected()
    .Setup<Task<Transition>>("ExecuteStepAsync", It.IsAny<CancellationToken>())
    .Returns(nonTargetedTransitionTask)
    .Verifiable();
stateMock.Protected()
    .Setup<Task<Transition>>("ExecuteStepAsync", It.IsAny<CancellationToken>())
    .Returns(Task.FromResult<Transition>(null))
    .Verifiable();
```

And had to do this:

```
stateMock.Protected()
    .Setup<Task<Transition>>("ExecuteStepAsync", It.IsAny<CancellationToken>())
    .Returns(nonTargetedTransitionTask)
    .Callback(() => stateMock.Protected()
        .Setup<Task<Transition>>("ExecuteStepAsync", It.IsAny<CancellationToken>())
        .Returns(nonTargetedTransitionTask)
        .Callback(() => stateMock.Protected()
            .Setup<Task<Transition>>("ExecuteStepAsync", It.IsAny<CancellationToken>())
            .Returns(Task.FromResult<Transition>(null))
            .Verifiable())
        .Verifiable())
    .Verifiable();
```

**Comment by stakx** [comment]
@paulomorgado, @s-limo - `SetupSequence` is now available in Moq 4.8.0.
